pagerduty/resource_pagerduty_event_orchestration_test.go

```diff
@@ -62,9 +62,16 @@ func TestAccPagerDutyEventOrchestration_Basic(t *testing.T) {
 			{
 				Config: testAccCheckPagerDutyEventOrchestrationConfigNameOnly(name),
 				Check: resource.ComposeTestCheckFunc(
-					testAccCheckPagerDutyEventOrchestrationExists("pagerduty_event_orchestration.nameonly"),
+					testAccCheckPagerDutyEventOrchestrationExists("pagerduty_event_orchestration.foo"),
```

Comment:
I modified the tests a bit to cover the following flow:
- create an orchestration that only has `name`
- update the orchestration to also have `description` and `team`
- update `name`, `description`, `team`
- delete `description` and `team`